src/mumble/MainWindow.h

```diff
@@ -280,6 +281,9 @@ class MainWindow : public QMainWindow, public MessageHandler, public Ui::MainWin
 		/// filename is discarded).
 		void updateImagePath(QString filepath) const;
 
+	signals:
+		void corkStream(const bool corked);
```

Comment:
I would give this signal a more verbose name. Something like `corkAudioStream`.

Also I'd change the parameter name  to `cork` instead of `corked` as I think the latter indicates that the one emitting the signal should provide the information whether the stream is currently corked or not. That might be just me though :)

And finally it'd be great, if you could add a Doxygen-comment to the signal describing what it is used for. Something like
```
/// This signal is used to ask the audio-backend to cork or un-cork the audio-stream
///
/// @param cork True for corking and false for un-corking
void corkStream(const bool corked);
```

Reply:
I decided to go with `corkAudioInputStream` instead, as the audio output is not controlled by this logic. Sound good?

I added some comments in 2d30973d1b5324dd4255bb54b34d6902ae7bc859. Let me know if you want me to trim them down at all.

Reply:
Yeah that's even better than my suggestion :+1:

Reply:
And the comments look great to me